### DISCUSSION TITLE
Storage File Upload Error

### DISCUSSION THREAD

**Discussion by GroovierHen** [supabase/supabase#7881] [Questions]
I'm uploading pictures to my storage bucket but I'm getting this error.
`Failed to upload test-pic-2.jpg: null value in column "id" of relation "objects" violates not-null constraint`

[error image](https://cdn.discordapp.com/attachments/995946390364618803/1001008458360967288/Screen_Shot_2022-07-24_at_11.09.56_PM.png)

**Comment by JSmithOner** [comment]
You may have a constraint that the field I should be not null. Maybe if you don't add an auto increment logic you need to manually  update this field which you maybe haven't

**Reply by GroovierHen** [reply]
I'm just uploading to my bucket from supabase panel. I don't understand which field it requires.

**Reply by JSmithOner** [reply]
Trying sending your question an issue it's maybe a bug.

**Comment by bnjmnt4n** [comment]
Hey @GroovierHen! Wanted to check if you've added any custom triggers or RLS policies which could possibly affect the storage tables? Otherwise, please send a support request so that we are able to debug this.

**Comment by flazouh** [comment]
I get the same error: 
<img width="477" alt="Screenshot 2023-05-12 at 01 28 33" src="https://github.com/supabase/supabase/assets/25705704/0c97127b-b2c1-4d41-89ef-d6c1a0f38d61">
